backend/functions/api/package.json

```diff
@@ -0,0 +1,13 @@
+{
+  "name": "cams-api",
+  "version": "1.0.0",
+  "description": "",
+  "main": "dist/backend/functions/api/*/*.function.js",
+  "scripts": {
+    "clean": "rm -rf dist/ node_modules/ *.zip",
```

Comment:
I'm curious about the difference between 
`rm -rf foo` and `rm -rf foo/`

Does the latter delete everything in the folder and leave the empty folder there?

Reply:
I just tested both and saw no difference.